Gems/Atom/Feature/Common/Code/Source/CommonSystemComponent.cpp

```diff
@@ -292,10 +296,13 @@ namespace AZ
             // setup handler for load pass template mappings
             m_loadTemplatesHandler = RPI::PassSystemInterface::OnReadyLoadTemplatesEvent::Handler([this]() { this->LoadPassTemplateMappings(); });
             RPI::PassSystemInterface::Get()->ConnectEvent(m_loadTemplatesHandler);
+            
+            m_modelReloaderSystem = AZStd::make_unique<ModelReloaderSystem>();
```

Comment:
uber nit: Every chunk of code in this function seems to have a comment, maybe add one there too.

Reply:
Will do.